### PR TITLE
Update README

### LINKED ISSUE #327: Update README

### PR DESCRIPTION
**Pull request by ianyong**
Resolves #327.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,39 @@
...
+Odyssey allows you to dream big while it does the nitty gritty for you: each challenge breaks a goal down into manageable, day-to-day tasks that increment slowly in difficulty and intensity.
+By providing you with concrete steps, you can be assured that they are on the right track to achieving your goal.
+
+What really makes Odyssey complete is its community aspect.
+Working towards a goal alone can lead us to feel lost and lonely, and having someone to guide us or even just to hold us accountable has been shown to increase our likelihood to follow through on what we commit to.
+This is why Odyssey makes it easy for you to hop on a challenge with your friends or share your progress, struggles and achievements within your friend circle and a wider community.
+
+Odyssey is also peppered with delightful, carefully designed details to make your in-app experience and overall journey enjoyable.
+With features ranging from a statistics column for those interested in measuring their progress to maps of unique lands that you can explore with your friends and collect mementos from, working towards your goals has never been breezier.
+
```
Comment (jialin7878) [inline README.md:24]: if u want can add ```<p align="center">  </p>``` here  (it's the slideshow of maps)

File: README.md [hunk 2]
```diff
@@ -1 +1,39 @@
...
+Start your odyssey today with [Odyssey](https://odyssey-app.com)!
+
+## Team
+
+| Member      | Role |
+| ----------- | ----------- |
+| Cai Jia Lin (A0205575A) | Full stack, UI, documentation |
+| Ian Yong Yew Chuang (A0135451M) | Full stack, deployment, documentation |
+| Sebastian Toh Shi Jian (A0196545R) | Full stack, deployment, documentation |
+| Wu Weiming (A0210627N) | Frontend, UI, documentation |
```
Comment (jialin7878) [inline README.md:35]: wm full stack also?

Comment (ianyong) [reply README.md:35]: Oh yea, I copied this from Hansel and didn't realise it's different. I'll update with what we have in our final report. Thanks for spotting!

File: README.md (hunk 1 above)
Comment (ianyong) [reply README.md:32]: Great idea! I've added the gifs from STePS coz they're too good, though I'm not too sure about their placement. Feel free to swap them around!

(Also note that GitHub does not render any CSS for security reasons so no fancy borders.)

**Review by ming-00 — APPROVED** [review]
Looks great, thanks! 😺

**Pull request merged**